apps/meteor/app/utils/lib/slashCommand.ts

```diff
@@ -30,6 +42,7 @@ export const slashCommands = {
 			providesPreview: Boolean(providesPreview),
 			previewer,
 			previewCallback,
+			appId,
```

Comment:
I didnt get why not only add this change here, why the whole 'refactor' was required?

Reply:
I think you misunderstood my point.
I have nothing against passing `appId` as a property of options.

But as your pr is marked as a Regression(and before as fix), I couldn't understand how this changes helped to fix the problem introduced. now that you explained that it has nothing to do with it the issue, it was just aesthetic, I can ignore this part